content/modules/redisgears/jvm/install.md

```diff
@@ -14,25 +14,12 @@ Before you can use RedisGears with the JVM, you need to install the RedisGears m
 
 1. Redis Enterprise v6.0.12 or later
 
-1. Created a Redis Enterprise cluster
+1. Created a Redis Enterprise cluster TODO link
 
-1. Added nodes to the cluster
+1. Added nodes to the cluster TODO link
 
-## Install RedisGears and the JVM plugin
+1. Install RedisGears and the JVM plugin TODO link to installation instructions
 
-If your cluster uses Redis Enterprise v6.0.12 or later, you only need to download the RedisGears package. The JVM plugin installs automatically when you make a `POST` request to the `/v2/modules` REST API endpoint.
```

Comment:
the change @rrelledge did on this PR (before @K-Jo changes) are correct, we should just move them to installing-redisgears.md page (I guess...)

Reply:
Yup, I see I forgot to move this

Reply:
I moved this info to the requested page.